### PR TITLE
Timer Feature

### PR DESCRIPTION
**Pull request by Anonuevo-Jheferson**
This is a Timer in HTML.

It has a Start, Pause, Resume and Stop buttons to control the text count.

Assigned reviewer: @alejandro-andreijames

### CODE REVIEW

File: timer.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta
+			name="viewport"
+			content="width=device-width, initial-scale=1.0"
```
Comment (Alejandro-AndreiJames) [inline timer.html]: @anonuevo-jheferson 
Suggestion:
Refer to the highlighted code.
You can just do this code instead to save code length:
`content="width=device-width, initial-scale=1.0"/>`

**Review by Alejandro-AndreiJames — CHANGES_REQUESTED** [review]
@anonuevo-jheferson 
I may suggest that you should rename the html code's file name, into "index.html".

The reason is that the Netlify uses index.html as the default file served when someone visits your site. If your main application is in a file named timer.html, you have two options:
1.	Rename timer.html to index.html: This is the simpler solution. By renaming the file, it becomes the default and automatically served by Netlify.
2.	Create a redirect rule in a _redirects file: This approach keeps timer.html but instructs Netlify to redirect visitors to that file. Here's how to create the redirect rule:
o	Create a new file named "_redirects" (without a file extension) in the root directory of your site.
o	Inside the "_redirects" file, add the following line:
`/* /timer.html 200`

This line tells Netlify to serve timer.html for any request made to your site (represented by /*). The 200 code specifies a successful response. Remember to include the "_redirects" file when deploying your site to Netlify. Netlify will pick up the file and apply the redirect rules accordingly.

File: timer.html (hunk 1 above)
Comment (Anonuevo-Jheferson) [reply timer.html]: > @anonuevo-jheferson Suggestion: Refer to the highlighted code. You can just do this code instead to save code length: `content="width=device-width, initial-scale=1.0"/>`

suggestion had been applied thank you!
https://github.com/PUP-BSIT/exercise-12-anonuevo-jheferson/blob/64ce7322d1d3fd99588eb758626268b7a69dc844/timer.html#L5

suggestion had been applied. renamed the html file - d6e511cb9776a11828772127436a2aefd06dae69

Comment (Alejandro-AndreiJames) [reply timer.html]: This is acknowledged. Thank you!

**Review by Alejandro-AndreiJames — APPROVED** [review]
This is good to go! If there are any concerns, then feel free to reach our team. Thank you.

**Pull request merged**